src/waterfall/polarplot.cpp

```diff
@@ -127,9 +129,10 @@ void PolarPlot::updateMouseColumnData()
     // Calculate grad from mouse position
     const QPoint deltaScaled(delta.x()*_image.width()/width(), delta.y()*_image.height()/height());
     int grad = static_cast<int>(atan2f(-deltaScaled.x(), deltaScaled.y())*rad2grad + 200) % 400;
+    _mouseSampleAngle = grad*grad2deg;
+    // Claculate mouse distance in meters
+    _mouseSampleDistance = 2*std::hypotf(delta.x(), delta.y())/std::max(width(), height())*_maxDistance*1e-3;
```

Comment:
Can you add a comment explaining this? What I got is that you are using the factor between the radius of the current mouse position and the full circle multiplied by the resolution in meters. But then why the 2* and why not `min()` instead of `max()`?

Reply:
The `2` comes from the division of the width|height, so it goes up as a multiplication, will add the equations to be clear.
The `std::` call was removed in the improved version.